processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -73,7 +76,13 @@ public LookupReferencesManager(LookupConfig lookupConfig, final @Json ObjectMapp
   @LifecycleStart
   public void start()
   {
-    synchronized (lock) {
+    try {
```

Comment:
This also holds the locks in an interruptible manner. This allows management tasks which have not started yet to be interrupted if something is holding a lock too long.